templates/types/streaming/nextjs/app/api/chat/stream-helper.ts

```diff
@@ -37,3 +37,17 @@ export function appendEventData(data: StreamData, title?: string) {
     },
   });
 }
+
+export function appendToolData(
+  data: StreamData,
+  toolCall: ToolCall,
+  toolOutput: ToolOutput,
+) {
+  data.appendMessageAnnotation({
+    type: "tools",
+    data: {
+      toolCall,
+      toolOutput,
+    } as any,
```

Comment:
is it a JSONValue then? (We should avoid any if possible)
```suggestion
    } as JSONValue,
```